pages/index.js

```diff
@@ -3,32 +3,55 @@ import Head from "next/head";
 import React, { useState, useEffect } from "react";
 import { btoa } from "abab";
 
-export async function getStaticProps() {
+function getUntappd() {
   const untappdApi = "https://business.untappd.com/api/v1";
   const auth = btoa(`${process.env.UNTAPPD_USER}:${process.env.UNTAPPD_TOKEN}`);
   const opts = { headers: { Authorization: `Basic ${auth}` } };
 
-  const [items, onDeck] = await Promise.all([
+  return Promise.all([
     fetch(`${untappdApi}/sections/90568/items`, opts),
     fetch(`${untappdApi}/sections/90575/items`, opts),
     fetch(`${untappdApi}/sections/90567/items`, opts),
   ])
     .then(([a, b, c]) => Promise.all([a.json(), b.json(), c.json()]))
     .then(([a, b, c]) => [[...a.items, ...b.items], c.items]);
+}
+
+function getWordPress() {
+  const wordPressApi = "http://localhost:8888/wp-json/wp/v2";
```

Comment:
Oh yeah, `localhost:8888` only exists inside your computer. In order for this to work on Netlify/in the build you’ll have to have a WordPress instance up and running somewhere on the web. Then you'll change this line to be like (just an example):

```ts
const wordPressApi = "https://cog-brews.wordpress.com/wp-json/wp/v2";
```

The name `cog-brews.wordpress.com` is just a stand in, it could be anything really, just as long as you can reach it on the internet.